hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -748,6 +750,72 @@ public Array getArray(int columnIndex) {
 		throw new UnsupportedOperationException();
 	}
 
+	public Array getArray(int columnIndex, JdbcType elementJdbcType) {
+		// TODO:  check object type and implement interface methods to convert
+		//
+		Object stringArray = row.getValue( columnIndex - 1 );
+		wasNull = stringArray == null;
+		if ( stringArray == null ) {
+			return null;
+		}
+		return new Array() {
+			@Override
+			public String getBaseTypeName() throws SQLException {
+				return elementJdbcType.getFriendlyName();
+			}
+
+			@Override
+			public int getBaseType() throws SQLException {
+				return elementJdbcType.getJdbcTypeCode();
+			}
+
+			@Override
+			public Object getArray() throws SQLException {
+				return stringArray;
+			}
+
+			@Override
+			public Object getArray(Map<String, Class<?>> map) throws SQLException {
+				return null;
```

Comment:
Unless it causes other issues, it would be better to throw an exception instead of returning `null`

Reply:
done